aspnet/fundamentals/index.rst

```diff
@@ -1,10 +1,11 @@
-Fundamentals
-============
+Fundamentals z
+===============
 
 .. toctree::
     :titlesonly:
 
     startup
+    choosing-the-right-dotnet
```

Comment:
Let's put this at the bottom of Fundamentals. My gut is that this article eventually needs to move out of the ASP.NET Core docs to be a general .NET article.